assets/scripts/store/reducers/settings.js

```diff
@@ -0,0 +1,28 @@
+import { SET_USER_SETTINGS } from '../actions'
+// Note: turning this constant on runs too much side-effect code too early.
```

Comment:
There's just too many things that depend on each other right now (another aspect of its past as a single JS everything-is-global file) -- perhaps centralizing constants in a few locations and exporting them from modules whose only purpose is to store constants is the right move. Anyway, I've updated the comment, hopefully it is clearer.